corpus/convert.py

```diff
@@ -12,6 +18,94 @@
 Path = setup_path(__package__)
 
 
+class CorpusReplaceOrthFromReferenceCorpus(Job):
+    """
+    Copies the orth tag from one corpus to another through matching segment names.
+    """
+
+    def __init__(self, bliss_corpus, reference_bliss_corpus):
+        """
+        :param bliss_corpus: Corpus in which the orth tag is to be replaced
+        :param reference_bliss_corpus: Corpus from which the orth tag replacement is taken
+        """
+        self.bliss_corpus = bliss_corpus
+        self.reference_bliss_corpus = reference_bliss_corpus
+
+        self.out_corpus = self.output_path("corpus.xml.gz")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        orth_c = corpus.Corpus()
+        orth_c.load(tk.uncached_path(self.reference_bliss_corpus))
```

Comment:
I just noticed you are using `tk.uncached_path` here, please change that to `.get_path()` here and for other occurances.

Reply:
I noticed in the `CorpusReplaceOrthFromTxtJob` it was also using `uncached_path` so I changed it there aswell in a different commit (to allow rollback). If its fine changed there aswell this should be resolved then.